CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(APPLE)
 	set_property(DIRECTORY
 		APPEND
 		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
```

Comment:
afaik cmake's set(...) command would need the FORCE argument to override -D params - this should be okay. And on OSX, install is required for any real use, as the executables in their build locations aren't really self-contained.